src/pidcontrollers/rate.hpp

```diff
@@ -167,6 +171,15 @@ namespace hf {
                 _IConstants[1] = gyroPitchI;
                 _DConstants[0] = gyroRollD;
                 _DConstants[1] = gyroPitchD;
+                
+                // Balance Mosquito 90
```

Comment:
This can be removed. We no longer require offsets for the rate controller. We can set them to 0 and leave them in case some need arises.